src/styles/stars-1.css

```diff
@@ -0,0 +1,3 @@
+.stars--1 > .stars__star:nth-child(1) {
+  background-image: url(../images/star-active.svg);
```

Comment:
There is no need to create a separate file for each stars modifier. Please delete them.
You can also combine the selectors with common style properties

Reply:
I had misunderstanding with this point "Each BEM block has its own separate file" so that is why I decided to seperate each stars modifier. I'll try to improve my code